src/world/Action/ActionLut.h

```diff
@@ -7,24 +7,43 @@ namespace Sapphire::World::Action
 {
   struct ActionEntry
   {
-    uint16_t potency;
-    uint16_t comboPotency;
-    uint16_t flankPotency;
-    uint16_t frontPotency;
-    uint16_t rearPotency;
-    uint16_t curePotency;
-    uint16_t restoreMPPercentage;
+    uint16_t damagePotency;
+    uint16_t damageComboPotency;
+    uint16_t damageDirectionalPotency;
+    uint16_t healPotency;
+    uint16_t selfHealPotency;
+    uint16_t selfStatus;
+    uint32_t selfStatusDuration;
+    uint16_t selfStatusParam;
+    uint16_t targetStatus;
+    uint32_t targetStatusDuration;
+    uint16_t targetStatusParam;
+    uint16_t gainMPPercentage;
+  };
+
+  struct StatusEffectEntry
+  {
+    uint32_t effectType;
+    int32_t effectValue1;
```

Comment:
what do these even mean? calling them all value is insanely obscure - if you're storing various kinds of data in there that have different meanings it makes much more sense to use a union